### PR TITLE
[display] UTM param handling

### PR DESCRIPTION
**Pull request by brendangkchan**
Full spec in ticket
https://ubiquity6.atlassian.net/browse/DL-1361

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -138,46 +138,113 @@ <h1 class="heading">Capture and Edit Reality<br></h1>
...
+          ? appendUTMQueryParams(APP_STORE_LINKS.IOS_WITH_UTM)
+          : APP_STORE_LINKS.IOS;
+      } else {
+        return utm_source
+          ? appendUTMQueryParams(APP_STORE_LINKS.ANDROID_WITH_UTM)
+          : APP_STORE_LINKS.ANDROID;
+      }
+    }
+
+    $(document).ready(function() {
```
Comment (brendangkchan) [inline index.html]: just fixed indenting and comment casing from L193-227

File: index.html [hunk 2]
```diff
@@ -138,46 +138,113 @@ <h1 class="heading">Capture and Edit Reality<br></h1>
...
+      // Stop the youtube video playing once we hide the popup model
       $('.modal').click(function(e) {
     		e.preventDefault();
         $('iframe').attr('src', $('iframe').attr('src'));
       });
 
-      //phone number input area
-      //submit button
+      // iOS App Store button click handler
```
Comment (brendangkchan) [inline index.html]: new click handlers for app store buttons

File: index.html [hunk 3]
```diff
@@ -138,46 +138,114 @@ <h1 class="heading">Capture and Edit Reality<br></h1>
...
-      autoHideDialCode: false,
-      nationalMode: false,
-      preferredCountries: ['us'],
-      separateDialCode: true,
-      utilsScript: "intl-tel-input/js/utils.js",
+    // We use different links for tracking and attribution purposes.
+    var APP_STORE_LINKS = {
+      IOS: 'https://displayland.app.link/MZRCzjpbz1',
+      ANDROID: 'https://displayland.app.link/Oh9yuuzbz1',
+      IOS_WITH_UTM: 'https://displayland.app.link/MZRCzjpbz1',
```
Comment (lhong375) [inline index.html]: why IOS and IOS_WITH_UTM are the same 🤔

File: index.html [hunk 4]
```diff
@@ -138,46 +138,114 @@ <h1 class="heading">Capture and Edit Reality<br></h1>
...
+        alert("close popup");
+    		$('.popup-bg').fadeOut();
+    	});
     });
-  });
-  $(document).ready(function() {
-  	// set unique id to videoplayer for the Webflow video element
-  	var src = $('#videoplayer').children('iframe').attr('src');
-  	// when object with class open-popup is clicked...
-  	$('.open-popup').click(function(e) {
```
Comment (lhong375) [inline index.html]: If my memory is working, these $(document).ready around '.open-popup' is not working. but I'm fine leaving it here, just fyi.

File: index.html (hunk 3 above)
Comment (brendangkchan) [reply index.html]: was a placeholder until I had branch login perms, just updated them now

**Pull request merged**